index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio MVOC</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>Welcome</h1>
+      <nav>
+        <a href="#">
+          <img src="/images/Icon-Menu.png" alt="hamburger-icon">
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2 class="name">
+          Hey there. I'm <br> Marco
+        </h2>
```

Comment:
- [ ] Kindly change this to `h1`. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.